### ISSUE TITLE
[CLI] bundle sourceMapURL should be relative to bundle file

### ISSUE THREAD

**Issue by clintwood** [esperantojs/esperanto#162]
When using the CLI with -b option the `sourceMapURL` is an absolute absolute path (I'm on Windows).
I assume this it should be a relative path to the `.js` source file. I.e.:
```
esperanto -b -s -t cjs -m -i 'main.js' -o ./build/bundle.js
// ./build/bundle.js file has
...
//# sourceMappingURL= /full/path/from/root/build/bundle.js.map

// and should be
...
//# sourceMappingURL=./bundle.js.map
```
This is due to [this line doing `path.resolve( options.output )` instead of `options.output`](https://github.com/esperantojs/esperanto/blob/master/bin/runEsperanto.js#L69)

**Comment by clintwood** [comment]
@Rich-Harris, additionally, source maps have `"sources": [ "<absolute/path/to/src>" ], ...` which on my Windows system looks something like `"../D:/absolute/path/to/src/file.js"`. I assume these also need to be relative to original source? I realise this specifying `"sourcesContent": [...]` may make these redundant, however, it is revealing to have absolute paths in any event.

**Comment by Rich-Harris** [comment]
Argh, sourcemaps tie my brain in knots! I *think* this is fixed if you use Rollup directly, or via 0.8 when it's released, so I'll close this as `wontfix`. If not please [raise an issue on Rollup](https://github.com/rollup/rollup/issues). Sorry for the crazy slow response
